tests/integration/index.ts

```diff
@@ -242,3 +242,308 @@ describe("Positive Cases", () => {
     });
   });
 });
+
+describe("Straight Cases For Single User Liquidation and healing", () => {
+  let fixture;
+  let Comptroller: Comptroller;
+  let vBNX: VToken;
+  let vBSW: VToken;
+  let BNX: MockToken;
+  let BSW: MockToken;
+  let acc1: string;
+  let acc2: string;
+  let PriceOracle: PriceOracle;
+  let vBSWPrice;
+  let vBNXPrice;
+
+  beforeEach(async () => {
+    ({ fixture } = await setupTest());
+    ({ Comptroller, vBNX, vBSW, BNX, BSW, acc1, acc2, PriceOracle } = fixture);
+  });
+  describe("Force Liquidation of user via Comptroller", () => {
+    const mintAmount = convertToUnit("1", 8);
+    let acc1Signer: Signer;
+    let acc2Signer: Signer;
+    const bswBorrowAmount = 1e4;
+    let result;
+
+    beforeEach(async () => {
+      acc1Signer = await ethers.getSigner(acc1);
+      acc2Signer = await ethers.getSigner(acc2);
+
+      await BNX.connect(acc2Signer).faucet(mintAmount);
+      await BNX.connect(acc2Signer).approve(vBNX.address, mintAmount);
+
+      // Fund 2nd account
+      await BSW.connect(acc1Signer).faucet(mintAmount);
+      await BSW.connect(acc1Signer).approve(vBSW.address, mintAmount);
+
+      await expect(vBNX.connect(acc2Signer).mint(mintAmount))
+        .to.emit(vBNX, "Mint")
+        .withArgs(acc2, mintAmount, mintAmount);
+      //borrow
+      //Supply WBTC to market from 2nd account
+      await expect(vBSW.connect(acc1Signer).mint(mintAmount))
+        .to.emit(vBSW, "Mint")
+        .withArgs(acc1, mintAmount, mintAmount);
+      //It should revert when try to borrow more than liquidity
+      await expect(vBSW.connect(acc2Signer).borrow(8e10)).to.be.revertedWithCustomError(
+        Comptroller,
+        "InsufficientLiquidity",
+      );
+      await expect(vBSW.connect(acc2Signer).borrow(bswBorrowAmount))
+        .to.emit(vBSW, "Borrow")
+        .withArgs(acc2, bswBorrowAmount, bswBorrowAmount, bswBorrowAmount);
+
+      vBNXPrice = Number(await PriceOracle.getUnderlyingPrice(vBNX.address)) / 10 ** 18;
```

Comment:
For safety and consistency should we use BigNumber here?

Is there a difference between the two before hooks? They could potentially use a fixture

Reply:
BigNumber is fixed
There are multiple tickets scenarios in one block and have slightly changes in each before.